media/css/firefox/fxos-nav.less

```diff
@@ -0,0 +1,576 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+// mixin for sprite images
+.fxosnav-sprite(@top) {
+    background-image: url(/media/img/firefox/os/nav-sprite.png);
+    background-position: 0 @top;
+
+    @media (-webkit-min-device-pixel-ratio: 1.5),
+       (                min-resolution: 1.5dppx),
+       (                min-resolution: 144dpi) {
+
+        .background-size(98px 115px);
+        background-image: url(/media/img/firefox/os/nav-sprite-high-res.png);
+    }
+}
+
+// big left hand logo
+.fxosnav-logo {
+    .fxosnav-sprite(0);
+    .image-replaced();
+    .transform(translateY(30px));
+    .transition(-webkit-transform 0.3s ease);
+    .transition(transform 0.3s ease);
```

Comment:
I'm not sure repeating this mixin actually works. The output is:
```
-webkit-transition: -webkit-transform 0.3s ease;
transition: -webkit-transform 0.3s ease;
-webkit-transition: transform 0.3s ease;
transition: transform 0.3s ease;
```
I'm not sure of the support overlap with `-webkit-transition` and `-webkit-transform` (if there's any version of webkit that supports one but not the other). As it is, browsers that only support `-webkit-transition` will try to apply it to `transform` which they may not support.

For proper full coverage you'd really need:
```
-webkit-transition: -webkit-transform 0.3s ease, transform 0.3s ease;
transition: -webkit-transform 0.3s ease, transform 0.3s ease;
```
But at this point I wonder if we need to worry much about old webkits for this kind of minor enhancement.